pkg/apis/build/v1alpha2/build_types.go

```diff
@@ -132,6 +132,7 @@ type BuildStatus struct {
 	corev1alpha1.Status    `json:",inline"`
 	BuildMetadata          corev1alpha1.BuildpackMetadataList `json:"buildMetadata,omitempty"`
 	Stack                  corev1alpha1.BuildStack            `json:"stack,omitempty"`
+	Lifecycle              ResolvedClusterLifecycle           // TODO: confirm
```

Comment:
Is this what we want? Or should we take a subset of the fields in `ResolvedClusterLifecycle`?